### ISSUE TITLE
Nice to have a switch for HDR, but it doesn't work properly in custom resolution mode

### ISSUE THREAD

**Issue by Doria-7** [moonlight-stream/moonlight-ios#498]
1. When the custom resolution is set, but the HDR switch is not turned on, it can be connected to the computer normally, and it can be displayed normally at the native resolution and ratio of the ipad. When the HDR switch is turned on, it cannot connect to the computer, and it prompts "The connection was terminated".

2. If it is in 1080p, 4k, it can be successfully turned on and connected, but the screen cannot fill the ipad screen, there will be black borders up and down.

3. models and systems：
    ipad pro 11 （ipad OS 15.4)
    PC  (windows 11 21H2)
    GeForce Experience version: 3.25.0.84
    Nvidia GPU driver: 512.15

**Comment by felipejfc** [comment]
I can reproduce on iPhone 12 Pro when trying to stream with HDR on and 2532x1170 resolution

**Comment by Doria-7** [comment]
I tried going back to an older GeForce driver (511.79), but still no success in HDR streaming. But I seem to have found a way to connect to the computer with the HDR switch turned on.
Cause a very accidental situation, I successfully streamed to the PC with the HDR switch turned on, but the resolution was forced to change. So I guess it may be that the PC desktop resolution is the same as the moonlight custom resolution, which causes the conflict to cause a bug that cannot be streamed. So I modified the custom resolution of moonlight (2387*1668) while keeping the PC desktop resolution unchanged (2388*1668). And then successfully streamed to the PC. The aspect ratio and size were normal, and the HDR display was also normal.
